### DISCUSSION TITLE
Infra requirement to setup RC FE and BE

### DISCUSSION THREAD

**Discussion by sudarshanswati** [Sunbird-RC/community#202] [Sunbird RC Technical Specification Feedback]
Please let me know the cloud infra requirement to set up both Frontend and backend

**Comment by sudarshanswati** [comment]
@pramodkvarma

**Comment by pramodkvarma** [comment]
@parthlawate @dileepbapat can you point Sujeet to it? If not, maybe help Sujeet document and do a PR on docs.

**Comment by parthlawate** [comment]
@sudarshanswati Here are the docs related to this https://docs.sunbirdrc.dev/use/infrastructure-requirements

**Reply by sudarshanswati** [reply]
@parthlawate I don't think this will help anyway. We need to understand what are all cloud components required for production setup. This document only contains one line "Kubernetes is recommended".

RC has multiple components like core, services, UI , Mobile etc. Can you please share the deployment requirement and instruction like it is mentioned for Sunbird LMS

**Comment by dileepbapat** [comment]
* For frontend it can be served from CDN alone or backed by static files served from Nginx (say small/medium instance).
* Backend services deployment config is [here](https://github.com/Sunbird-RC/sunbird-rc-core/tree/main/infra) can be deployed on any kubernetes cluster.  Load balancer, CDN requirements might be cloud specific. Number of nodes in the cluster and type of instance depends on usecase and cloud provider also.
* Datastores - Postgres 2 instanced (for replication), Elastic search cluster (if used)

Updated https://docs.sunbirdrc.dev/use/infrastructure-requirements with typical/minimal production configuration used other implementations.

**Reply by sudarshanswati** [reply]
Thanks a lot @dileepbapat  this is really helpful
